backend/scraper/__init__.py

```diff
@@ -0,0 +1,93 @@
+"""Scrape research papers from the Semantic Scholar API and put them into the database."""
+import semanticscholar as ss
+import requests
+
+def scrape():
+    """Scrape research papers from the Semantic Scholar API"""
+    # Determine the ids of all relevant research papers.
+    papers = get_all_papers()
+    for paperId in papers:
+        # Get all relevant information for the paper: id, title, abstract, year
+        paper = ss.paper(paperId)
+        paperTitle = paper['title']
+        paperAbstract = paper['abstract']
+        paperYear = paper['year']
+        # Put the given paper into the database.
+        post_paper(paperId, paperTitle, paperAbstract, paperYear)
+
+        # Get all relevant information for the author: id, name
+        authors = paper['authors']
+        for author in authors:
+            authorId = author['authorId']
+            authorName = author['name']
+            # Put the given author and writing relation into the database.
+            post_author(authorId, authorName)
+            post_write(paperId, authorId)
+
+        # Get all references
+        citations = paper['citations']
+        if citations == []:
+            citations = paper['references']
+        for citation in citations:
+            citationId = citation['paperId']
+            post_reference(paperId, citationId)
+
+
+def get_all_papers():
+    """Determine the ids of all relevant research papers."""
+    authorIds = [143993045, 143990000, 144000000, 144100000, 144122431]
```

Comment:
It would be extraordinarily nice if you could write the authors names in a comment so that it's easier to reason about these lines in the future. Also a mapping between names and ID helps if we want to change one of the initial authors